easybuild/easyblocks/g/gromacs.py

```diff
@@ -41,7 +41,8 @@
 from easybuild.tools.build_log import EasyBuildError
 from easybuild.tools.modules import get_software_root
 from easybuild.tools.systemtools import get_platform_name
-
+from easybuild.tools.systemtools import get_shared_lib_ext
+import easybuild.tools.toolchain as toolchain
```

Comment:
please move the `import` one up below the other `import` (keep them alphabetical), and collapse the two for `systemtools`:

```python
from easybuild.tools.systemtools import get_platform_name , get_shared_lib_ext
```